CS483/CS483/Kartaclysm/Components/Abilities/ComponentBedazzleAbility.cpp

```diff
@@ -107,7 +107,7 @@ namespace Kartaclysm
 		{
 			if (m_pConditions->CanActivate() &&
 				pAIDriver->DistanceToClosestOpponent() <= 2.0f &&
-				rand() % 8 == 0)
+				rand() % 30 == 0)
```

Comment:
A 1/30 chance every frame still seems pretty high, considering we should be targeting 60 FPS.  If you think it's good, then that's fine, but I have a feeling it will still be activated pretty quickly once we get the frame rate sorted out.